app-template/working-with-template.md

```diff
@@ -10,30 +10,47 @@ keywords: Docker, application template, Application Designer,
 
 ## Overview
 
-Docker Template is a CLI plugin that introduces a top-level `docker template` command that allows users to create new Docker applications by using a library of templates. There are two types of templates — service templates and application templates.
+Docker Template is a CLI plugin that introduces a top-level `docker template`
+command that allows users to create new Docker applications by using a library
+of templates. There are two types of templates — service templates and
+application templates.
 
-A _service template_ is a container image that generates code and contains the metadata associated with the image.
+A _service template_ is a container image that generates code and contains the
+metadata associated with the image.
 
-- The container image takes `/run/configuration` mounted file as input to generate assets such as code, Dockerfile, and `docker-compose.yaml` for a given service, and writes the output to the `/project` mounted folder.
+- The container image takes `/run/configuration` mounted file as input to
+  generate assets such as code, Dockerfile, and `docker-compose.yaml` for a
+given service, and writes the output to the `/project` mounted folder.
 
-- The metadata file that describes the service template is called the service definition.  It contains the name of the service, description, and available parameters such as ports, volumes, etc. For a complete list of parameters that are allowed, see [Docker Template API reference](/ee/app-template/api-reference).
+- The metadata file that describes the service template is called the service
+  definition.  It contains the name of the service, description, and available
+parameters such as ports, volumes, etc. For a complete list of parameters that
+are allowed, see [Docker Template API
+reference](/ee/app-template/api-reference).
 
-An _application template_ is a collection of one or more service templates. An application template generates a Dockerfile per service and only one Compose file for the entire application, aggregating all services.
+An _application template_ is a collection of one or more service templates. An
+application template generates a Dockerfile per service and only one Compose
+file for the entire application, aggregating all services.
 
 ## Create a custom service template
 
-A Docker template contains a predefined set of service and application templates. To create a custom template based on your requirements, you must complete the following steps:
+A Docker template contains a predefined set of service and application
+templates. To create a custom template based on your requirements, you must
+complete the following steps:
 
 1. Create a service container image
 2. Create the service template definition
 3. Add the service template to the library
 4. Share the service template
 
-### Create a service container image
+### A service container image
```

Comment:
Not sure why this was changed, the previous titled seems to make more sense, and is more consistent with the other sections.

Reply:
Yup, makes sense. The title has been updated in https://github.com/docker/docker.github.io/pull/9537.